src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -110,15 +119,29 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
 
   const hasUpcomingEvents = futureUpcoming && futureUpcoming.length > 0;
 
+  const hasRadioStations = radioStations && radioStations.length;
+  const isSoundCloud =
+    !!hasRadioStations && RegExp("soundcloud").test(radioStations![0]);
+
   const sound = useMemo(
     () =>
-      radioStations && radioStations.length
+      radioStations && radioStations.length && !isSoundCloud
         ? new Audio(radioStations[0])
         : undefined,
-    [radioStations]
+    [isSoundCloud, radioStations]
   );
 
-  const { volume, setVolume } = useRadio(sound);
+  const [isRadioPlaying, setIsRadioPlaying] = useState(false);
+  const { volume, setVolume } = useRadio(isRadioPlaying, sound);
+
+  const soundcloudIframeRef = useRef<HTMLIFrameElement>(null);
+
+  // Append script to <body>
+  useEffect(() => {
+    if (isSoundCloud) {
+      loadScript("https://w.soundcloud.com/player/api.js");
+    }
+  }, [isSoundCloud]);
```

Comment:
I believe we only needed this when we wanted to control the playback, but I don't think we're actually using it anymore?

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/928/commits/e8fd26ba30e9505036b0f1cda1afcec36d4ab4bf